python/ray/tests/test_memory_pressure.py

```diff
@@ -148,6 +168,24 @@ def test_memory_pressure_kill_task(ray_with_memory_monitor):
     )
 
 
+@pytest.mark.skipif(
+    sys.platform != "linux" and sys.platform != "linux2",
+    reason="memory monitor only on linux currently",
+)
+def test_task_crash_after_raylet_dead_throws_node_died_error(ray_with_memory_monitor):
```

Comment:
This test probably belongs somewhere else since it doesn't have to do with memory monitoring. Maybe in a test_failure suite.